release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -104,9 +104,7 @@ With {product-title} {product-version}, running `oc describe` on an image now re
 
 This release includes an enhancement to the user experience for the control plane machine set on Microsoft Azure clusters.
 
-For Azure clusters that are installed with {product-title} version 4.13, you are no longer required to create a control plane machine set custom resource (CR). Instead, an inactive CR is generated for the cluster and can be activated after you verify that the values in the CR are correct for your control plane machines.
-
-Azure clusters that are upgraded to version 4.13 from an earlier version still require xref:../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-creating-cr_cpmso-getting-started[manual CR creation].
+For Azure clusters that are installed with or upgraded to {product-title} version 4.13, you are no longer required to create a control plane machine set custom resource (CR). Instead, an inactive CR is generated for the cluster and can be activated after you verify that the values in the CR are correct for your control plane machines.
```

Comment:
I think this fixes it, if I am understanding correctly now. @odvarkadaniel PTAL :)

Reply:
Nice, LGTM!